test/README.md

```diff
@@ -168,3 +182,20 @@ Windows vs non-windows portability env vars:
     SEP:           \ or / (required)
     OBJ:          .obj or .o (required)
     EXE:          .exe or <null> (required)
+
+BASH Tests
+------------------------------
+Along with environment variables defined by the Makefile (see above), there is another a set of variables that can be used inside BASH tests that are defined in `tools/exported_vars.h`.
```

Comment:
- `.h` -> `.sh`
- BTW technically these variables are also exported in the postscript tests.

Reply:
but it looks like they are "defined" in tools/exported_vars.h

Reply:
Point was that the file ending is .sh -> https://github.com/dlang/dmd/blob/master/test/tools/exported_vars.sh

Reply:
yeah got that one

Reply:
oh woops...I mispelled it again in my response! :)